modules/007-registrypackages/images/containerd-centos/scripts/install

```diff
@@ -14,14 +14,30 @@
 # limitations under the License.
 
 set -Eeo pipefail
-PACKAGES_TO_INSTALL=""
-if ! rpm --quiet -q "$(rpm -qip container-selinux.x86_64.rpm | grep Name | awk '{print $3}')"; then
-  PACKAGES_TO_INSTALL="container-selinux.x86_64.rpm"
+packages_to_install=""
+package_container_selinux="container-selinux"
+package_file_container_selinux="${package}.x86_64.rpm"
```

Comment:
i do not see `package` variable, maybe here should be `package_container_selinux`?

Reply:
yes i is error, thanks